db/migrations/0003_player_created_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-10-30 17:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_guild_description'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='player',
+            name='created_at',
+            field=models.DateField(auto_now=True),
```

Comment:
Consider using `auto_now_add=True` instead of `auto_now=True` for the `created_at` field. `auto_now_add` sets the field to the current date when the model instance is created, which is more appropriate for a creation timestamp.